src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,23 @@
+package calculator;
+
+public class StringCalculator {
+
```

Comment:
```suggestion
    private final Delimiter delimiter;
    private final CalculateStrategy calculateStrategy;

    public StringCalculator(Delimiter delimiter, CalculateStrategy calculateStrategy) {
        this.delimiter = delimiter;
        this.calculateStrategy = calculateStrategy;
    }
```

Delimiter, CalculateStrategy 의 구현체를  add 할 때 생성하는 것이 아니라 
위와 같이 `StringCalculator` 객체를 생성시에 인터페이스를 주입 받을 수 있으면 구현체의 변경이 자유롭겠죠?